starknet_devnet/server.py

```diff
@@ -102,9 +121,8 @@ def main():
     set_gas_price(args)
 
     try:
-        meinheld.listen((args.host, args.port))
         print(f" * Listening on http://{args.host}:{args.port}/ (Press CTRL+C to quit)")
```

Comment:
Instead of removing this line, could we remove all the gunicorn [INFO] logs? They appear on startup and on SIGINT.